transit_odp/timetables/tests/test_csv.py

```diff
@@ -42,18 +42,16 @@ def test_service_in_bods_but_not_in_otc():
         assert row["Published Status"] == "Published"
         assert row["OTC Status"] == "Unregistered"
         assert row["Scope Status"] == "In Scope"
-        assert row["DQ Score"] == "33%"
-        assert row["BODS Compliant"] == "NO"
-        assert row["XML Filename"] == txc_file_attributes.filename
-        assert row["Last Updated Date"] == dataset.live_revision.published_at
+        assert row["XML:Filename"] == txc_file_attributes.filename
```

Comment:
While we assert the values - is it possible to verify the order of the columns? either along with this line or as a separate test case?